STYLE_GUIDE.md

```diff
@@ -118,6 +128,36 @@ A procedure is a series of numbered steps that a user follows to complete a spec
 - Parallel language constructions
 - Consistent formatting
 
+Replace pointer-specific language with device-agnostic language to accommodate readers with disabilities and users of various input methods and devices, including the pointer, keyboard, and touch screens.
+
+For example, instead of the term *click*, which is pointer-specific, use *choose*, which is more generic and device-agnostic. However, when the generic language makes it difficult to understand the instructions (for example, in the case of opening a context menu), you can include pointer-specific hints. Use your judgment. If you have a question, ask your editor.
+
+Use the following language to describe UI interactions:
+
+- Use *choose* to describe moving to a UI component such as a tab or pane or taking action on a button or menu.
+- Use *select* to describe picking a user-specific resource or enabling one of several options. Contrast with *clear* to turn off previously selected options.
+- Use *press* to describe single key or key combination entries that users would perform on a keyboard.
+- Use *enter* to describe information that users add using a keyboard.
+- Do not use *hit* or *strike*.
+
+The following table provides examples of language to be used to describe interactions with UI elements.
+
+| UI element | Language | Example |
+| :--------- | :--------| :-------|
+| Menu| On the *menu* menu, choose *command*.| On the **Edit** menu, choose **Copy**.|
```

Comment:
Add spaces between the letter and the start/end of each table column. This could prevent formatting errors later.

Reply:
Done.